site/content/en/docs/tasks/operations/standalone-deployment-mode.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Standalone Deployment Mode"
+---
+
+{{% alert title="Notice" color="warning" %}}
+
+Standalone mode is an experimental feature, please **DO NOT** use it in production.
+
+{{% /alert %}}
+
+Envoy Gateway also supports running in standalone mode. In this mode, Envoy Gateway
+does not need to rely on Kubernetes and can be deployed directly on bare metal or virtual machines.
+
+Currently, Envoy Gateway only support the file provider and the host infrastructure provider combinations.
+
+- The file provider will configure the Envoy Gateway to get all gateway-api resources from file system.
+- The host infrastructure provider will configure the Envoy Gateway to deploy one Envoy Proxy as a host process.
+
+## Quick Start
+
+In this quick-start, we will run Envoy Gateway in standalone mode with the file provider
+and the host infrastructure provider.
+
+### Prerequisites
+
+Create a local directory just for testing:
+
+```shell
+mkdir -p /tmp/envoy-gateway-test
+```
+
+Download the Envoy Gateway binary from v1.2.x release.
+
+### Create Certificates
+
+All runners in Envoy Gateway are using TLS connection, so create these TLS certificates locally to 
+ensure the Envoy Gateway works properly.
+
+```shell
+envoy-gateway certgen --local
+```
+
+### Start Envoy Gateway
+
+Start Envoy Gateway by the following command:
+
+```shell
+envoy-gateway server --config-path standalone.yaml
+```
+
+with `standalone.yaml` configuration:
+
+```yaml
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: EnvoyGateway
+gateway:
+  controllerName: gateway.envoyproxy.io/gatewayclass-controller
+provider:
+  type: Custom
+  custom:
+    resource:
+      type: File
+      file:
+        paths: ["/tmp/envoy-gateway-test"]
+    infrastructure:
+      type: Host
+      host: {}
+logging:
+  level:
+    default: info
+extensionApis:
+  enableBackend: true
+```
+
+As you can see, we have enabled the [Backend][] API, this API will be used to represent our local endpoints.
+
+Since Envoy Gateway is running in the standalone mode, we highly recommend using the [Backend][] API instead of 
```

Comment:
Probably just remove this line - it makes no sense to use K8s services and EndpointSlice in the host mode.

Reply:
sgtm